vector/src/main/java/im/vector/app/features/form/FormEditTextItem.kt

```diff
@@ -75,13 +76,8 @@ abstract class FormEditTextItem : VectorEpoxyModel<FormEditTextItem.Holder>() {
         holder.textInputLayout.error = errorMessage
         holder.textInputLayout.endIconMode = endIconMode ?: TextInputLayout.END_ICON_NONE
 
-        if (holder.view.isAttachedToWindow) {
-            // the view is attached to the window
-            // So it is a rebind of new data and you could ignore it assuming this is text that was already inputted into the view.
-            // Downside is if you ever wanted to programmatically change the content of the edit text while it is on screen you would not be able to
-        } else {
-            holder.textInputEditText.setText(value)
-        }
+        holder.textInputEditText.setValueOnce(value, holder)
```

Comment:
It looks a bit weird, I would set the Extension to VectorEpoxyHolder, and pass the TextInputEditText, and the value as parameter, to write it like that: `holder.setValueOnce(textInputEditText, value)`, but it's maybe not better, WDYT?

Reply:
Done